File: apps/api-orders/create-zip.py
import zipfile
import os

with zipfile.ZipFile('v2.10.0-s3-documents.zip', 'w', zipfile.ZIP_DEFLATED) as z:
    for root, dirs, files in os.walk('deploy'):
        for f in files:
            path = os.path.join(root, f)
            arc = path.replace('deploy/', '', 1).replace('deploy\\', '', 1)
            z.write(path, arc)

print('ZIP created successfully')

File: apps/api-orders/create-zip.js
const fs = require('fs');
const path = require('path');
const archiver = require('archiver');

const output = fs.createWriteStream(path.join(__dirname, 'deploy-v2.18.0-node.zip'));
const archive = archiver('zip', { zlib: { level: 9 } });

output.on('close', () => {
  console.log('Archive created: ' + archive.pointer() + ' bytes');
});

archive.on('error', (err) => {
  throw err;
});

archive.pipe(output);

// Add directory contents with Unix paths
archive.directory('deploy-package-v2.18/', false);

archive.finalize();
